DecipherNow/gm-ui-components
# fix: blank page when clicking anchor links in styles story

Resolves #417 

To test
- `npm start`
- view Styles story
- click Color, Typography and Layout - they should bring you to the correct headers
